app/js/trouble.js

```diff
@@ -1,22 +1,28 @@
-'use strict';
+var jQuery;

```

Comment:
To clear the linting error referencing jQuery, go into the configuration options on jshint and turn on jQuery in the "assume" section. This allows jshint to "assume" the existence of a globally declared jQuery variable.